corehq/apps/userreports/tests/test_transforms.py

```diff
@@ -21,6 +21,34 @@ def testInvalidType(self):
             })
 
 
+class NumberFormatTransformTest(SimpleTestCase):
+
+    def setUp(self):
+        self.transform = TransformFactory.get_transform({
+            "type": "number_format",
+            "format_string": "{0:.0f}"
+        }).get_transform_function()
+
+    def test_int(self):
+        self.assertEqual('11', self.transform(11))
+
+    def test_decimal(self):
+        self.assertEqual('11', self.transform(11.23))
+
+    def test_strings(self):
+        tests = (
+            ('11', '11'),
+            ('11.23', '11'),
+            ('notanumber', 'notanumber'),
+            ('', ''),
+        )
```

Comment:
nit: you can use `generate_cases` for this kind of thing: https://github.com/dimagi/commcare-hq/blob/1e184803217ee121cf43adb24e50e16d9062befe/corehq/util/test_utils.py#L231